articles/cms/joomla/installation.md

```diff
@@ -1,12 +1,21 @@
 ---
-description: How to install a Joomla application with Auth0.
+description: How to install Auth0 onto your Joomla platform.
 ---
 
-
 # Joomla Installation
 
-1. Download the latest release of the Joomla extension from [auth0-joomla](https://github.com/auth0/auth0-joomla/releases).
-2. Access the Joomla administrator interface and select *Extension Manager* from the **Extensions** dropdown menu.
-3. Select *Choose File* under *Upload & Install Joomla Extension* and select the downloaded zip file.
-4. Click *Upload & Install*.
-5. [Setup the plugin](/cms/joomla/configuration)
+[Download](https://github.com/auth0/auth0-joomla/releases) the latest release of the Auth0-Joomla extension.
```

Comment:
Maybe add a numbered list to depict steps more clearly? ie `1. [Download](https://github.com/auth0/auth0-joomla/releases) the latest release of the Auth0-Joomla extension.`